2014-03-17 15:45:48 Error: [MissingControllerException] Controller class DiariocamposController could not be found.
Exception Attributes: array (
  'class' => 'DiariocamposController',
  'plugin' => NULL,
)
Request URL: /si_apropiacion/diariocampos/add
Stack Trace:
#0 C:\xampp\htdocs\si_apropiacion\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-03-17 15:46:06 Error: [MissingControllerException] Controller class ReunionesController could not be found.
Exception Attributes: array (
  'class' => 'ReunionesController',
  'plugin' => NULL,
)
Request URL: /si_apropiacion/reuniones
Stack Trace:
#0 C:\xampp\htdocs\si_apropiacion\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-03-17 15:46:09 Error: [MissingControllerException] Controller class ReunionController could not be found.
Exception Attributes: array (
  'class' => 'ReunionController',
  'plugin' => NULL,
)
Request URL: /si_apropiacion/reunion
Stack Trace:
#0 C:\xampp\htdocs\si_apropiacion\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-03-17 15:46:18 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\si_apropiacion\Controller\ReunionsController.php, line 58]
2014-03-17 15:46:18 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /si_apropiacion/reunions/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-03-17 15:46:24 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\si_apropiacion\Controller\ReunionsController.php, line 58]
2014-03-17 15:46:24 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /si_apropiacion/reunions/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-03-17 15:46:31 Error: [NotFoundException] Invalid reunion
Request URL: /si_apropiacion/reunions/edit
Stack Trace:
#0 [internal function]: ReunionsController->edit()
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ReunionsController), Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(ReunionsController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\si_apropiacion\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-17 15:48:12 Error: [MissingControllerException] Controller class AcompanamientoController could not be found.
Exception Attributes: array (
  'class' => 'AcompanamientoController',
  'plugin' => NULL,
)
Request URL: /si_apropiacion/acompanamiento
Stack Trace:
#0 C:\xampp\htdocs\si_apropiacion\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-03-17 17:05:16 Error: [MissingControllerException] Controller class AcompannamientoController could not be found.
Exception Attributes: array (
  'class' => 'AcompannamientoController',
  'plugin' => NULL,
)
Request URL: /si_apropiacion/acompannamiento
Stack Trace:
#0 C:\xampp\htdocs\si_apropiacion\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-03-17 17:44:27 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\si_apropiacion\Controller\UsersController.php, line 58]
2014-03-17 17:44:27 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /si_apropiacion/users/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-03-17 17:44:45 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\si_apropiacion\Controller\UsersController.php, line 58]
2014-03-17 17:44:45 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /si_apropiacion/users/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-03-17 17:52:01 Error: [MissingControllerException] Controller class DiariocamposController could not be found.
Exception Attributes: array (
  'class' => 'DiariocamposController',
  'plugin' => NULL,
)
Request URL: /si_apropiacion/diariocampos
Stack Trace:
#0 C:\xampp\htdocs\si_apropiacion\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-03-17 17:52:04 Error: [MissingControllerException] Controller class DiariocamposController could not be found.
Exception Attributes: array (
  'class' => 'DiariocamposController',
  'plugin' => NULL,
)
Request URL: /si_apropiacion/diariocampos/add
Stack Trace:
#0 C:\xampp\htdocs\si_apropiacion\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-03-18 17:47:29 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\appropriation_is\Controller\UsersController.php, line 58]
2014-03-18 17:47:29 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /appropriation_is/users/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-03-18 17:47:52 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\SiteTypes\add.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\SiteTypes\\add.ctp',
)
Request URL: /appropriation_is/site_types/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(SiteTypesController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 17:49:30 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`appropriation_is`.`people`, CONSTRAINT `people_ibfk_1` FOREIGN KEY (`id_person`) REFERENCES `agents` (`person_id`) ON DELETE CASCADE ON UPDATE CASCADE)
Request URL: /appropriation_is/people/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `ap...', Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(1004): DboSource->execute('INSERT INTO `ap...')
#3 C:\xampp\htdocs\cake\lib\Cake\Model\Model.php(1832): DboSource->create(Object(Person), Array, Array)
#4 C:\xampp\htdocs\appropriation_is\Controller\PeopleController.php(51): Model->save(Array)
#5 [internal function]: PeopleController->add()
#6 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(PeopleController), Array)
#7 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(PeopleController), Object(CakeRequest), Object(CakeResponse))
#9 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-03-18 17:50:17 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`appropriation_is`.`people`, CONSTRAINT `people_ibfk_1` FOREIGN KEY (`id_person`) REFERENCES `agents` (`person_id`) ON DELETE CASCADE ON UPDATE CASCADE)
Request URL: /appropriation_is/people/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `ap...', Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(1004): DboSource->execute('INSERT INTO `ap...')
#3 C:\xampp\htdocs\cake\lib\Cake\Model\Model.php(1832): DboSource->create(Object(Person), Array, Array)
#4 C:\xampp\htdocs\appropriation_is\Controller\PeopleController.php(51): Model->save(Array)
#5 [internal function]: PeopleController->add()
#6 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(PeopleController), Array)
#7 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(PeopleController), Object(CakeRequest), Object(CakeResponse))
#9 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-03-18 17:50:53 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`appropriation_is`.`people`, CONSTRAINT `people_ibfk_1` FOREIGN KEY (`id_person`) REFERENCES `agents` (`person_id`) ON DELETE CASCADE ON UPDATE CASCADE)
Request URL: /appropriation_is/people/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `ap...', Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(1004): DboSource->execute('INSERT INTO `ap...')
#3 C:\xampp\htdocs\cake\lib\Cake\Model\Model.php(1832): DboSource->create(Object(Person), Array, Array)
#4 C:\xampp\htdocs\appropriation_is\Controller\PeopleController.php(51): Model->save(Array)
#5 [internal function]: PeopleController->add()
#6 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(PeopleController), Array)
#7 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(PeopleController), Object(CakeRequest), Object(CakeResponse))
#9 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-03-18 17:54:06 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'PRIMARY'
Request URL: /appropriation_is/people/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `ap...', Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php(1004): DboSource->execute('INSERT INTO `ap...')
#3 C:\xampp\htdocs\cake\lib\Cake\Model\Model.php(1832): DboSource->create(Object(Person), Array, Array)
#4 C:\xampp\htdocs\appropriation_is\Controller\PeopleController.php(51): Model->save(Array)
#5 [internal function]: PeopleController->add()
#6 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(PeopleController), Array)
#7 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(PeopleController), Object(CakeRequest), Object(CakeResponse))
#9 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-03-18 17:56:19 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\appropriation_is\Controller\UsersController.php, line 58]
2014-03-18 17:56:19 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /appropriation_is/users/add
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 58, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-03-18 19:39:33 Error: [MissingControllerException] Controller class USERSaddController could not be found.
Exception Attributes: array (
  'class' => 'USERSaddController',
  'plugin' => NULL,
)
Request URL: /appropriation_is/USERSadd
Stack Trace:
#0 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-03-18 19:39:43 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\Users\\login.ctp',
)
Request URL: /appropriation_is/users/login
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 19:39:52 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\Users\\login.ctp',
)
Request URL: /appropriation_is/users/login
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 19:42:14 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\Users\\login.ctp',
)
Request URL: /appropriation_is/users/login
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 19:42:24 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\Users\\login.ctp',
)
Request URL: /appropriation_is/users/login
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 19:46:06 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\Users\\login.ctp',
)
Request URL: /appropriation_is/users/login
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 19:46:15 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\Users\\login.ctp',
)
Request URL: /appropriation_is/users/login
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 19:46:20 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\Users\\login.ctp',
)
Request URL: /appropriation_is/users/login
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 19:46:29 Error: [MissingViewException] View file "C:\xampp\htdocs\appropriation_is\View\Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\appropriation_is\\View\\Users\\login.ctp',
)
Request URL: /appropriation_is/users/login
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(457): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(952): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-03-18 20:17:45 Error: [MissingActionException] Action UsersController::display() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'display',
)
Request URL: /appropriation_is/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-03-18 20:17:51 Error: [MissingActionException] Action UsersController::display() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'display',
)
Request URL: /appropriation_is/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-03-18 20:18:30 Error: [MissingActionException] Action UsersController::display() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'display',
)
Request URL: /appropriation_is/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\appropriation_is\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
